src/Dialect/ONNX/Transforms/Decompose.cpp

```diff
@@ -1439,6 +1500,15 @@ void onnx_mlir::getDecomposeONNXToONNXPatterns(
   patterns.insert<SoftmaxCrossEntropyPattern>(context);
   patterns.insert<SumToAddPattern>(context);
 
+  if (!onnx_mlir::decomposeOpsInONNX.empty()) {
+    for (const auto &op : onnx_mlir::decomposeOpsInONNX) {
+        llvm::outs() << "Decomposing ONNX operation: " << op << "\n";
```

Comment:
Please remove this `llvm::outs()` or you can change to using `LLVM_DEBUG(llvm::dbgs() << DEBUG_TYPE << "Decomposing ONNX operation: " << op << "\n"` instead.